### PR TITLE
Add Chakra Card component

### LINKED ISSUE #1024: Add Chakra Card Component
Now that we have upgraded our next/chakra/react we can add new chakra components.

This issue is to add Card, CardHeader, CardBody, CardFooter

https://chakra-ui.com/docs/components/card

### PR DESCRIPTION
**Pull request by iron3oxide**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

### New Feature Submission:

- [x] Does your submission pass the tests? 
- [x] Have you linted your code locally prior to submission?

This PR adds the Chakra Card component, along with CardHeader, CardBody and CardFooter implementations. Since there are only tests for the somewhat special `pc.cond()` layot component, I couldn't really figure out what and how to best test this, coverage is luckily still >80%. Docs have to be added to pcweb if I'm not mistaken, I'll have a look at how to best do that.

Closes #1024

### CODE REVIEW

**Comment by Lendemor** [comment]
I would suggest to use the same names as the tag for the components : 
`Card`, `CardHeader`, `CardBody`, `CardFooter`
and 
`card`, `card_header`,`card_body`,`card_footer` for the shortcuts.

File: pynecone/components/layout/card.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            footer (Optional[Component]): The footer of the Card.
+            props: The properties to be passed to the component.
+
+        Returns:
+            The `create()` method returns a Card object.
+        """
+        children = []
+        param_to_component_class = ((header, Chead), (body, Cbody), (footer, Cfoot))
+
+        for tup in param_to_component_class:
```
Comment (Lendemor) [inline pynecone/components/layout/card.py:85]:
```
for param, component_class in param_to_component_class:
```

Comment (iron3oxide) [reply pynecone/components/layout/card.py]: Didn't know you could do this, just added it.

**Comment by iron3oxide** [comment]
> I would suggest to use the same names as the tag for the components : `Card`, `CardHeader`, `CardBody`, `CardFooter` and `card`, `card_header`,`card_body`,`card_footer` for the shortcuts.

I took inspiration from how it's done in `Table`, figured the brevity would be more user friendly. But since everything apart from `card` will probably only be used sparingly, it makes sense to favour non-abbreviated naming.

**Comment by Lendemor** [comment]
Looking good, I checked the "require documentation update" in the checklist, you can remove the two other unchecked line since they are not relevant.

**Comment by iron3oxide** [comment]
Done. Is there any guide on how to add component docs?

**Comment by Lendemor** [comment]
If you want to add the documentation yourself, you can make a PR for it on [pcweb](https://github.com/pynecone-io/pcweb).

Otherwise we will do it later.

**Pull request merged**